wordpress-mobile/WordPress-iOS
# HACK week: Local feature flag cleanup

Part of pcdRpT-3HX-p2

## How to test
- Make sure the app builds
- Make sure tests pass
- Instead of manual testing, go through each commit 🙇‍♀️ 

## Regression Notes
1. Potential unintended areas of impact
feature flags

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:


UI Changes testing checklist:
